Classes/Resource/AssetProcessing.php

```diff
@@ -104,6 +114,7 @@ public function processFile(FileProcessingService $fileProcessingService, Driver
         // Update existing processed file
         $processedFile->updateProperties(
             [
+                BynderDriver::KEY => true,
```

Comment:
As the field in the `tables.sql` is hardcode. just set it here also direct instead of using the constant